composer.json

```diff
@@ -36,7 +36,7 @@
     "oat-sa/extension-tao-community": "10.3.1",
     "oat-sa/extension-tao-funcacl": "7.2.2",
     "oat-sa/extension-tao-dac-simple": "7.8.0",
-    "oat-sa/extension-tao-itemqti": "29.21.5",
+    "oat-sa/extension-tao-itemqti": "29.22.0",
```

Comment:
@rgomeztao @jsconan Can you please confirm that the version of tao-core used to rebundle was the same used on may release? 

https://github.com/oat-sa/extension-tao-itemqti/pull/2334/files

Reply:
The version of taoQtiitem is immediately following the one that was part of the initial May release.
<img width="278" alt="image" src="https://user-images.githubusercontent.com/1500098/231440472-eee28420-3292-4a68-8b45-d9114bc16671.png">

It has been built using the GitHub action, the tao-core is therefore not exactly the same version than in the May release. However, this should not do any difference: the version of tao-core was most probably `51.11.2`, while the this is `51.11.1` for May, there is no impact on the CSS nor the JS.

Regarding the code change on taoItemQti is mostly about CSS and a tiny change on the JS, and only the bundle for taoQtiItem is impacted, no matter what is around on tao-core.

I don't see any issue here in this particular context.

Reply:
Ok, thanks for the clarification. My concern was that when developing + bundling the original development, a different tao-core version was used. 

All clear.